client-cli/src/command.rs

```diff
@@ -196,7 +196,6 @@ impl Command {
 
     fn get_balance<T: WalletClient>(wallet_client: T, name: &str) -> Result<()> {
         let passphrase = ask_passphrase()?;
-        wallet_client.sync()?;
```

Comment:
why removed sync ?

Reply:
Most of the client logic will change in future. Now that new have view keys, client will have to maintain view key of current user somewhere and then pass it to `sync()`. Currently, we don't have that.